aspnetcore/mvc/views/partial.md

```diff
@@ -1,173 +1,303 @@
+::: moniker range=">= aspnetcore-2.1"
+
+A partial view is a *.cshtml* markup file maintained within the *Views* folder (MVC) or *Pages* folder (Razor Pages).
+
+In ASP.NET Core MVC, a controller's <xref:Microsoft.AspNetCore.Mvc.ViewResult> is capable of returning either a view or a partial view. An analogous capability is planned for Razor Pages in ASP.NET Core 2.2. In Razor Pages, a <xref:Microsoft.AspNetCore.Mvc.RazorPages.PageModel> can return a <xref:Microsoft.AspNetCore.Mvc.PartialViewResult>. Referencing and rendering partial views is described in the [Reference a partial view](#reference-a-partial-view) section.
+
+Unlike MVC view or page rendering, a partial view doesn't run *_ViewStart.cshtml*. For more information on *_ViewStart.cshtml*, see <xref:mvc/views/layout>.
 
-As a convention, partial view file names often begin with `_`. This naming convention isn't a requirement, but it helps to visually differentiate partial views from regular views.
+Partial view file names often begin with an underscore (`_`). This naming convention isn't required, but it helps to visually differentiate partial views from views and pages. When the file name starts with an underscore, Razor Pages doesn't process the markup file as a Razor Pages page, even when the file's markup includes the `@page` directive.
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-2.1"
+
+A partial view is a *.cshtml* markup file maintained within the *Views* folder.
+
+A controller's <xref:Microsoft.AspNetCore.Mvc.ViewResult> is capable of returning either a view or a partial view.
+
+Unlike MVC view rendering, a partial view doesn't run *_ViewStart.cshtml*. For more information on *_ViewStart.cshtml*, see <xref:mvc/views/layout>.
+
+Partial view file names often begin with an underscore (`_`). This naming convention isn't required, but it helps to visually differentiate partial views from views.
+
+::: moniker-end
 
 ## Reference a partial view
 
-Within a view page, there are several ways to render a partial view. The best practice is to use asynchronous rendering.
+::: moniker range=">= aspnetcore-2.1"
+
+Within a markup file, there are several ways to reference a partial view. We recommend that apps use one of the asynchronous rendering approaches:
+
+* [Partial Tag Helper](#partial-tag-helper)
+* [Asynchronous HTML Helper](#asynchronous-html-helper)
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-2.1"
+
+Within a markup file, there are two ways to reference a partial view. We recommend that apps use the [Asynchronous HTML Helper](#asynchronous-html-helper).
```

Comment:
It isn't obvious what the 2nd way is. I know it's the Synchronous HTML Helper option, but that may not be obvious to the reader.